### ISSUE TITLE
Plugin fails when JSON payload includes key with leading underscore

### ISSUE THREAD

**Issue by kmarkley** [FlyingDiver/Indigo-Shims#15]
Error:
```
   MQTT Shims Error                Error in plugin execution runConcurrentThread:

Traceback (most recent call last):
  File "plugin.py", line 107, in runConcurrentThread
  File "plugin.py", line 132, in processMessages
  File "plugin.py", line 416, in update
StandardError: LowLevelBadParameterError -- illegal character in XML tag name or value
```
Example payload:
```
{
"state_string": "Operational"
"_timestamp": 1597520731
"_event": "PrinterStateChanged"
"state_id": "OPERATIONAL"
}
```

**Comment by FlyingDiver** [comment]
Try https://github.com/FlyingDiver/Indigo-Shims/releases/tag/0.3.5

**Comment by kmarkley** [comment]
That seems to work, thanks.

I now have new issue, however.  The payload includes keys that are usually numeric, but are sometimes null.  In this example printTime, printTimeLeft, and others are initially null before the print physically begins, after which they become real numbers.

Example payload:
```
{"_timestamp": 1597529996, "progress": 0, "printer_data": {"progress": {"completion": null, "printTimeLeftOrigin": null, "printTime": null, "printTimeLeft": null, "filepos": null}, "state": {"text": "Operational", "flags": {"cancelling": false, "paused": false, "operational": true, "pausing": false, "printing": false, "resuming": false, "sdReady": true, "error": false, "ready": true, "finishing": false, "closedOrError": false}}, "currentZ": null, "job": {"averagePrintTime": null, "lastPrintTime": null, "user": "pi", "file": {"origin": "local", "name": "tray_sm_0.2mm_PLA_MK3S_7h4m-TEST-2.gcode", "date": 1597506981, "path": "tray_sm_0.2mm_PLA_MK3S_7h4m-TEST-2.gcode", "display": "tray sm_0.2mm_PLA_MK3S_7h4m-TEST-2.gcode", "size": 7046139}, "estimatedPrintTime": 15356.916729460274, "filament": {"tool0": {"volume": 99.13513212812876, "length": 41215.598529984934}}}, "offsets": {}}, "location": "local", "path": "tray_sm_0.2mm_PLA_MK3S_7h4m-TEST-2.gcode"}
```
Error message:
```   MQTT Shims Error                Error in plugin execution runConcurrentThread:

Traceback (most recent call last):
  File "plugin.py", line 115, in runConcurrentThread
  File "plugin.py", line 139, in processMessages
  File "plugin.py", line 734, in update
TypeError: state value type must be bool, int, real, or string

   MQTT Shims Error                plugin runConcurrentThread function returned or failed (will attempt again in 10 seconds)
```

Could these states remain blank or zero when null?

**Comment by FlyingDiver** [comment]
This issue fixed in 0.3.5.

**Comment by FlyingDiver** [comment]
@kmarkley See new issue.
